duckduckgo/duckduckgo-autofill
# feat: support Windows

**Reviewer:**
**Asana:** https://app.asana.com/0/0/1202122822813612/f

## Description

1) Adds support for Windows
2) Clean up event handlers for form submissions (see second commit)

## Steps to test

[ ] - Windows team sign off
[x] - Regular cross-platform testing
[ ] - MacOS catalina testing (is this required?)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201926832890712
